scripts/lib.sh

```diff
@@ -40,17 +40,21 @@ sudo-askpass() {
 init-docker() {
     # Need to start docker if it was freshly installed or updated
     # You will know that Docker is ready for devservices when the icon on the menu bar stops flashing
-    if query-mac && [ ! -f /Library/PrivilegedHelperTools/com.docker.vmnetd ]; then
+    if query-mac && ! require docker && [ -d "/Applications/Docker.app" ]; then
```

Comment:
I'm going back to the original checks from the bootstrap repo.

`[ ! -f /Library/PrivilegedHelperTools/com.docker.vmnetd ]` is always true since I don't have permission to check if it's there:
```
❯ [ ! -f /Library/PrivilegedHelperTools/com.docker.vmnetd ]; echo $?  
0
❯ ls -l /Library/PrivilegedHelperTools/com.docker.vmnetd 
ls: /Library/PrivilegedHelperTools/com.docker.vmnetd: Permission denied
❯ sudo ls -l /Library/PrivilegedHelperTools/com.docker.vmnetd 
Password:
-r-xr--r--  1 root  wheel  2979888 14 May 16:04 /Library/PrivilegedHelperTools/com.docker.vmnetd
```